.github/ISSUE_TEMPLATE/staging-review.md

```diff
@@ -24,10 +24,25 @@ assignees: andreahewitt-odd, meganhkelley
 - Access information for the tool or feature in staging _**do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository**_
 - Link to your product test cases/test plan in TestRail with test execution logs ([learn more](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/qa-artifacts.md#test-plan)) _for QA practice area review_
 - Link to your [Coverage for References](https://dsvavsp.testrail.io/index.php?/reports/view/12) and [Summary(Defects)](https://dsvavsp.testrail.io/index.php?/reports/view/14) reports in TestRail _for QA practice area review_
-- Link to your accessibility test cases:
```

Comment:
I am concerned about removing these links. I took great pains to include multiple ways for teams to document their pre-staging review accessibility checks. This includes screen shots and explicit definitions of done for each task. I'd ask that we keep these links, and add your steps below.

It does raise a good question: Is TestRail and/or the Markdown template the best place to include steps for teams to take, or should those steps be moved into other documents?

Reply:
Teams need time to get up to speed in TestRail and using the accessibility test cases Markdown template. At present, it is blocking teams from submitting staging review requests and impacting launch timelines. We can provide different options for teams, if that's appropriate, or use the QA test cases. Ideally, whatever helps teams ship.

Certainly, if teams created accessibility test cases, those should be linked. On the staging review template, I hoped to help teams understand what is required — at present they do not know they need to do browser scans, etc.

Reply:
We pair-programmed to fix this. 🤜 🤛 💥 ❗